User-Help/NewUserBlurb.md

```diff
@@ -0,0 +1,11 @@
+<!--- Local Intro Blurb -->
+Welcome to the Writing Codidact community! This is the place for those interested in the craft of writing, editing, and publishing. We welcome questions about all kinds and genres of writing: fiction, poetry, technical documentation, journalism, scriptwriting, non-fiction, essays, and more. For contributing guidelines, check out our site [FAQ's](https://writing.codidact.com/about). 
+
+<!--- Global Intro Blurb -->
+Your contributions make Codidact better! Got questions? Click the "ask" button. Able to offer answers or advice? Use the Lottery button to view a cross section of recent posts, or search for your favorite topics. As you browse, vote on content that stands out - it's how we improve. 
+
+To learn more about the Codidact family of communities and our sister sites, check out [our home page](https://codidact.com). 
+
+<!--- Transition Annoucement -->
+If you have an account on Writing Stack Exchange, check out [this question](https://writing.codidact.com/questions/39219#answer-39220) for instructions on how to claim your questions and answers with you account. 
```

Comment:
Let's replace this with direct instructions:

If you have an account on (site.SE), you can claim your imported questions and answers here. Edit your profile to add the link to your SE profile.  You will now own those posts here on our site, just as if you had posted them directly.